lib/grunt.js

```diff
@@ -40,10 +46,13 @@ module.exports = function(grunt) {
       htmlTemplate: './templates/html.hbs',
       html: true,
       rename: function(path) {
-        path = path.replace(svgRootDir, '');
+        // We just want the filename, so split on backslashes and pop the last one.
+        let fileName = path.split('/').pop();
```

Comment:
[basename](https://nodejs.org/api/path.html#path_path_basename_p_ext) is a bit more robust for this sort of thing.